hoth_api.yaml

```diff
@@ -402,9 +402,44 @@ components:
         templatePayload:
           type: object
           oneOf:
+            - $ref: "#/components/schemas/EvalPlan"
             - $ref: "#/components/schemas/DescriptionOfWork"
             - $ref: "#/components/schemas/IndependentGovernmentCostEstimate"
-    IndependentGovernmentCostEstimate:
+   EvalPlan:
+     description: Provides the basis of evaluation for an Aquisition Package
+     type: object
+     properties:
+        source_selection:
```

Comment:
More of a general question, but do we want to shift to using `camelCase` in the HOTH spect to match our models in Lambda?
I think I have a branch where I was started to update the HOTH spec to be camelCase for the DoW and IGCE, but never opened up a PR.

Reply:
There seems to be a mixture currently.  GenerateDocumentRequest and CspResponse have camelCase properties, with snake_case used in many other places.

UI team seems to use snake_case in their data models, and ServiceNow internals use snake_case.  That may explain the tendancy to lean that way.

I prefer consistency.

Reply:
Per the team discussion, the short term is to progress with `camelCase`.

Reply:
Agreed, per team discussion.  This API spec may be inconsistent in the short term, but we'll begin to lean toward use of `camelCase` property names.  A significant driver of that decision is due to [Middy](https://middy.js.org/) validator use in AWS lambdas.  In the long term this API spec and surrounding implementation should use `camelCase`.